python/tchannel/tornado/peer.py

```diff
@@ -418,6 +440,14 @@ def __init__(self, peer, service, **kwargs):
         self.peer = peer
         self.service = service
         self.parent_tracing = kwargs.get('parent_tracing', None)
+        self.headers = {
+            'as': kwargs.get('arg_scheme', None) or ArgScheme.default,
+            're': kwargs.get('retry', None) or Retry.default,
```

Comment:
Just add these (incl. parent_tracing) as construct arguments with default values of `None`